assets/js/blocks/store-notices/block.json

```diff
@@ -6,7 +6,14 @@
 	"category": "woocommerce",
 	"keywords": [ "WooCommerce" ],
 	"supports": {
-		"multiple": false
+		"multiple": false,
+		"align": [ "wide", "full" ]
```

Comment:
We need to update the `render` method of `./src/BlockTypes/StoreNotices.php`  to make the alignment work on the front end.

Reply:
I didn't realise earlier there's a separate `get_align_class_and_style` method for that. I applied the change to:
- Store Notices
- Breadcrumbs

It works fine when the Classic Template is replaced by blocks. They preserve the alignment. But I found a bug that when the alignments are changed it's starts to become unpredictable. I was trying to find a pattern when and what happens, but I don't see one. So I'm going to create an issue and describe the most I can.

Reply:
I created an issue: https://github.com/woocommerce/woocommerce-blocks/issues/8336. I believe it's not a blocker and also confirmed it's not related to the newly implemented blocks, but it seems more global.

Reply:
Nice catch!